conf/svcomp23.json

```diff
@@ -67,6 +67,7 @@
         "mallocWrappers",
         "noRecursiveIntervals",
         "enums",
+        "congruence",
```

Comment:
Maybe let's not activated it in the configuration on master yet, until we have an estimate of the impact (In Manuel's thesis there was a negative impact, but that was largely due to `Error(verify)` which should be fixed now).

Other than that, it looks good to merge.

Reply:
I removed the setting from `svcomp23.json`.